realm/realm-library/src/androidTest/java/io/realm/BulkInsertTests.java

```diff
@@ -538,7 +538,7 @@ public void testObjectNull() {
         try {
             realm.insert(nullObject);
             fail("Should trigger NullPointerException");
```

Comment:
update text to "IllegalArgumentException" or remove it. It's not very useful to have anyway :-)